packages/extension/src/language-client/client.ts

```diff
@@ -218,8 +228,8 @@ export class SQLToolsLanguageClient implements ILanguageClient {
     );
     if (!r) return;
     await Wspc.getConfiguration(EXT_CONFIG_NAMESPACE).update('useNodeRuntime', true, ConfigurationTarget.Global);
-    try { await Wspc.getConfiguration(EXT_CONFIG_NAMESPACE).update('useNodeRuntime', true, ConfigurationTarget.Workspace) } catch(e) {}
-    try { await Wspc.getConfiguration(EXT_CONFIG_NAMESPACE).update('useNodeRuntime', true, ConfigurationTarget.WorkspaceFolder) } catch(e) {}
+    try { await Wspc.getConfiguration(EXT_CONFIG_NAMESPACE).update('useNodeRuntime', true, ConfigurationTarget.Workspace) } catch (e) { }
+    try { await Wspc.getConfiguration(EXT_CONFIG_NAMESPACE).update('useNodeRuntime', true, ConfigurationTarget.WorkspaceFolder) } catch (e) { }
     const res = await window.showInformationMessage(
       `\'${EXT_NAMESPACE}.useNodeRuntime\' enabled. You must reload VSCode to take effect.`, 'Reload now');
```

Comment:
```suggestion
      `\'${EXT_NAMESPACE}.useNodeRuntime\' enabled. You must reload VS Code to take effect.`, 'Reload now');
```

Official way of writing the product name is with a space between "VS" and "Code". Let's align with this as we find occurrences.